source/get-started/bare-metal-install-server.rst

```diff
@@ -468,6 +468,100 @@ be configured post-installation using the ``cryptsetup`` tool.
 
       :guilabel:`Confirm` is only highlighted if passphrases match.
 
+Manage User
+===========
+
+Add New User
+------------
+
+#. In Required Options, select :guilabel:`Manage User`.
+
+#. Select :guilabel:`Add New User` as shown in Figure 18.
+
+   .. figure:: /_figures/bare-metal-install-server/bare-metal-install-server-18.png
+      :scale: 100%
+      :alt: Add New User, User Name
+
+      Figure 18: Add New User
+
+#. Optional: Enter a :guilabel:`User Name`.
+
+   .. note:
+
+      The User Name must be alphanumeric and can include spaces, commas, underscores or hyphens. Maximum length is 64 characters.
+
+   .. figure:: /_figures/bare-metal-install-server/bare-metal-install-server-19.png
+      :scale: 100%
+      :alt: User Name
+
+      Figure 19: User Name
+
+#. Enter a :guilabel:`Login`.
+
+   .. note::
+
+      The User Login must be alphanumeric and can include hyphens and underscores. Maximum length is 31 characters.
+
+#. Enter a :guilabel:`Password`.
+
+   .. note:
+
+      Minimum length is 8 characters. Maximum length is 255 characters.
+
+#. In :guilabel:`Confirm`, enter the same password.
+
+#. The :guilabel:`Administrator` checkbox is selected by default.
+   This assures a secure boot immediately after installation.
```

Comment:
```suggestion
 .. note::	

      Selecting Administrator enables sudo privileges for the user. At least one user must be assigned as an Administrator for the installation to proceed.
```

Reply:
I was also trying to eliminate this wording you had on line 514 "This assures a secure boot immediately after installation."  "Secure boot" has a specific meaning that might be misinterpreted.
If you want some text there, we should rephrase that something like "Defining an administrator user account allows the secure locking of the `root` account." or just drop the line which is what I had intended.